src/EnergyPlus/SingleDuct.cc

```diff
@@ -6015,6 +6015,95 @@ namespace SingleDuct {
         }
     }
 
+    void setATMixerSizingProperties(int const &inletATMixerIndex, // index to ATMixer at inlet of zone equipment
+                                    int const &controlledZoneNum, // controlled zone number
+                                    int const &curZoneEqNum       // current zone equipment being simulated
+    )
+    {
+        if (inletATMixerIndex == 0) return;
+        if (controlledZoneNum == 0) return;
+        if (curZoneEqNum == 0) return;
+        if (SingleDuct::SysATMixer(inletATMixerIndex).MixerType != DataHVACGlobals::ATMixer_InletSide) return;
+
+        // proceed to set ATMixer properties used for sizing coils
+        int airLoopIndex = // find air loop associated with ATMixer
+            DataZoneEquipment::ZoneEquipConfig(controlledZoneNum).InletNodeAirLoopNum(SingleDuct::SysATMixer(inletATMixerIndex).CtrlZoneInNodeIndex);
+        bool SizingDesRunThisAirSys = false;                               // Sizing:System object found flag
+        CheckThisAirSystemForSizing(airLoopIndex, SizingDesRunThisAirSys); // check for Sizing:System object
+        if (SizingDesRunThisAirSys) {
```

Comment:
@mjwitte would you review the math here to make sure I didn't make a mistake. I see one already. OA Frac should be based on air loop OA vs main loop flow. So this one is wrong since a single air loop can serve multiple zone equipment.

    Real64 OutAirFrac = ZoneEqSizing(curZoneEqNum).ATMixerVolFlow / FinalSysSizing(airLoopIndex).DesMainVolFlow;

Reply:
@Myoldmopar it looks like I'll be fixing other things so feel free to add to the list.

Reply:
Geeez, this is just woeful. I don't know if it's the way clang is formatting the code or if I was just getting tired. But this is bad in many ways. I'm fixing it now. Are there any other comments other than here?